cmd/plasmacli/subcmd/eth/prove.go

```diff
@@ -76,28 +69,51 @@ var proveCmd = &cobra.Command{
 
 // Returns transaction results for given position
 // Trusts connected full node
-func getProof(ctx context.CLIContext, addr ethcmn.Address, position plasma.Position) (*tm.ResultTx, []byte, error) {
-	// query for the output
-	key := append(addr.Bytes(), position.Bytes()...)
-	res, err := ctx.QueryStore(key, "utxo")
+func getProof(position plasma.Position) (*tm.ResultTx, []byte, error) {
+	ctx := context.NewCLIContext().WithTrustNode(true)
```

Comment:
I think ctx should be a parameter and `WithTrustNode` should not be explicitly set. `Trust-Node` can be set via flag/commandline/env/config.toml

Reply:
I can merge in my config PR first so that the changes are in effect before merging this into develop